src/main/resources/config/application.yml

```diff
@@ -54,6 +54,8 @@ management:
     health:
         mail:
             enabled: false # When using the MailService, configure an SMTP server and set this to true
+        binders:
```

Comment:
does it work with dev profile, where kafka not used?

Reply:
yes, you are right, my fault